students/src/main/java/com/example/students/service/StudentService.java

```diff
@@ -15,5 +15,51 @@
 @Service
 @RequiredArgsConstructor
 public class StudentService {
+    private final StudentRepository studentRepository;
+    private final StudentClient client;
+    StudentMapper studentMapper = new StudentMapper();
+    public StudentFullName getStudentFullName (Integer id) {
+        RestTemplate restTemplate = new RestTemplate();
+        StudentFullName fullName = new StudentFullName();
 
+        String resourceURL = String.format("http://name:8085/name/%d", id);// service: first-name
```

Comment:
extract this string into application.yml

Reply:
neeed meet